saleor/product/models.py

```diff
@@ -407,6 +417,15 @@ def annotate_quantities(self):
             ),
         )
 
+    def prefetch_variant_updated(self):
```

Comment:
Would be good to keep some consistency here. You have `prefetched_for_webhook` and here `prefetch_variant_updated`. I don't think that everyone will connect that this method is prefetch for webhook with name  PRODUCT_VARIANT_UPDATED`